app/api/localStorage/index.js

```diff
@@ -82,6 +83,20 @@ export default class LocalStorageApi {
     storageKeys.URI_SCHEME_ACCEPTANCE
   );
 
+  // ========== Level Complexity ========== //
+  getComplexityLevel: void => Promise<?string> = () => getLocalItem(
```

Comment:
it may be good to change all the `string` types in this PR to an enum. You can do something like this

```
export const ComplexityLevels = Object.freeze({
    Simple: 'simple,
    Advanced: 'advanced',
});
export type ComplexityLevelsT = $Values<typeof ComplexityLevels>;